fuel-core-interfaces/Cargo.toml

```diff
@@ -15,6 +15,7 @@ anyhow = "1.0"
 async-trait = "0.1"
 chrono = { version = "0.4" }
 derive_more = { version = "0.99" }
+fuel-crypto = { version = "0.6", default-features = false }
```

Comment:
This is actually re-exported via fuel-vm -> fuel-tx -> fuel-crypto

Reply:
Fixed, rust-analyzer didn't find it.